source/services/ApplicationVersionService.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "../helpers/ApiRequest";
+
+import VERSION_STATUS from "../types/VersionStatusTypes";
+
+interface GetApplicationVersionStatus {
+  status: VERSION_STATUS;
+  updateUrl: string;
+}
+const getApplicationVersionStatus = async (): Promise<
+  GetApplicationVersionStatus | Record<string, unknown>
+> => {
+  const response = await get("/version");
+
+  if (response.status !== 200) {
+    return {
+      // NOTE: we assume things are kind of great when AWS doesnt respond nice...
+      status: VERSION_STATUS.UPDATE_OPTIONAL,
+      updateUrl: "",
+    };
+  }
+
+  return response?.data?.data.attributes || {};
```

Comment:
I think `null` is a better fallback than `{}` as we can then the response against `null` and by that almost guarantee that the response is valid otherwise.